frontend/server/src/Controllers/Course.php

```diff
@@ -2439,7 +2439,9 @@ public static function apiAssignmentDetails(\OmegaUp\Request $r): array {
                 $nominationStatus['language'] = \OmegaUp\Controllers\Problem::getProblemStatement(
                     $problem['alias'],
                     $problem['commit'],
-                    \OmegaUp\Controllers\Identity::getPreferredLanguage($r)
+                    \OmegaUp\Controllers\Identity::getPreferredLanguage(
+                        $r->identity
```

Comment:
esto debería ser

```suggestion
                        self::resolveTargetIdentity($r)
```

no?

Reply:
Tienes razón, se modifica.